src/components/Divider.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Divider = ({color, style, ...other}) => (
+  <hr
+    style={{
+      borderTopColor: 'rgba(0, 0, 0, .12)',
+      ...style,
+    }}
```

Comment:
Careful of default browser styles--might wanna go full `<style jsx />` on this bish